docs/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Dependencies
+/node_modules
+
+# Production
+/build
+
+# Generated files
+.docusaurus
+.cache-loader
+
+# Misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Generated api documentation
+/docs/99-api/
+!/docs/99-api/_category_.yml
```

Comment:
Add a comment why this is needed, so it is obvious to future readers

Reply:
Added a comment.